vtex/ab-tester
# Make ab-tester use only new ABTesting API

Just remove part of code that uses old API.

#### How should this be manually tested?
Verify that A/B Testing still working. Also check `Metadata`'s buckets that stores A/B Test info.
